bk://bk.phunnypharm.org/ieee1394-2.6
bcollins@debian.org|ChangeSet|20040621132834|56241 bcollins

# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/06/21 09:28:34-04:00 bcollins@debian.org 
#   ieee1394: Fix PCILynx bus resets.
#   
#   Signed-off-by: Jody McIntyre <linux1394@modernduck.com>
#   Signed-off-by: Ben Collins <bcollins@debian.org>
# 
# drivers/ieee1394/pcilynx.c
#   2004/06/21 09:27:00-04:00 bcollins@debian.org +2 -2
#   Fix PCILynx bus resets.
#   
#   Signed-off-by: Jody McIntyre <linux1394@modernduck.com>
#   Signed-off-by: Ben Collins <bcollins@debian.org>
# 
diff -Nru a/drivers/ieee1394/pcilynx.c b/drivers/ieee1394/pcilynx.c
--- a/drivers/ieee1394/pcilynx.c	2004-06-28 19:18:16 -07:00
+++ b/drivers/ieee1394/pcilynx.c	2004-06-28 19:18:16 -07:00
@@ -500,7 +500,7 @@
         pcl.async_error_next = PCL_NEXT_INVALID;
         pcl.pcl_status = 0;
         pcl.buffer[0].control = packet->speed_code << 14 | packet->header_size;
-#ifdef __BIG_ENDIAN
+#ifndef __BIG_ENDIAN
         pcl.buffer[0].control |= PCL_BIGENDIAN;
 #endif
         pcl.buffer[0].pointer = d->header_dma;
@@ -1697,7 +1697,7 @@
         pcl.async_error_next = PCL_NEXT_INVALID;
 
         pcl.buffer[0].control = PCL_CMD_RCV | 16;
-#ifdef __BIG_ENDIAN
+#ifndef __BIG_ENDIAN
 	pcl.buffer[0].control |= PCL_BIGENDIAN;
 #endif
 	pcl.buffer[1].control = PCL_LAST_BUFF | 4080;
